clang/lib/Driver/Driver.cpp

```diff
@@ -9732,9 +9739,9 @@ const ToolChain &Driver::getOffloadingDeviceToolChain(const ArgList &Args,
         switch (Target.getArch()) {
           case llvm::Triple::spir:
           case llvm::Triple::spir64:
-            TC = std::make_unique<toolchains::SYCLToolChain>(
-              *this, Target, HostTC, Args);
-            break;
+          TC = std::make_unique<toolchains::SYCLToolChain>(
+              *this, Target, HostTC, Args, DeviceTraitsMacrosArgs);
+          break;
```

Comment:
Formatting update is not consistent - was this something that was found by clang-format?

Reply:
It was, but I agree that this is not correct. Sometimes I don't understand clang-format. Thank you for finding this!